documentation/add/07-sqw_redesign.md

```diff
@@ -120,7 +146,7 @@ The same `get_data(name)` method can be used to provide access to the "standard"
 (3): existing binary data format must be extended to include additional "number of columns" field (&ge;9) information and the file read/write routines updated to read from an array of `n` columns.
 
 
-### Image
+### Image (`SQWDnDBase` class in more details)
```

Comment:
May need a bit more detail. Access to the image is available via abstract methods but the concrete image is stored in a DNDBase class either for a DnD or and SQW

Reply:
I've expanded the sentence (I believe, this left from Nick), but would not dare to expand it further. It may go forever.